### ISSUE TITLE
Names for Training courses

### ISSUE THREAD

**Issue by glilienthal** [KSP-RO/RP-0#907]
I got two landing capsules, the smaller and the bigger one. The courses for both of them are called "Landing Proficiency" Maybe https://github.com/KSP-RO/RP-0/blob/Developmental/Source/Crew/TrainingDatabase.cs is a little bit overeager in simplifying the names?

**Comment by RoboRay** [comment]
Have you determined if they are, in fact, different courses?  Build a test craft with both capsules, take just one of the courses, and see if you are allowed to board only one or both of the capsules.

**Comment by glilienthal** [comment]
No, I did not check that. Will do, and come back.

**Comment by pap1723** [comment]
@glilienthal any update?

**Comment by glilienthal** [comment]
not yet.

**Comment by glilienthal** [comment]
I checked it: And amazingly, it doesn't matter at all which "landing" course one takes. So the bug is _a lot_ smaller, and is just that the same course is shown twice.

**Comment by glilienthal** [comment]
There doesn't seem to be a time set for mission training for the landers. 

Does anyone understand the syntax of the below?

```
	// Apollo
	Apollo = 200, MatureCapsules								// 600
		bluedog-Apollo-Block2-Capsule = Apollo
		FASAApollo-CM = Apollo
		SSTU-SC-B-CM = Apollo
		Mark1-2Pod = Apollo		
	Apollo-Mission = 60
		
		// Advanced Apollo										   650
		SSTU-SC-B-CMX = 50, Apollo
		bluedog-Apollo-Block3-Capsule = 50, Apollo
		
	// LEM
	FASA-Gemini-Lander-Pod = 100, Landing
	landerCabinSmall = 100, Landing
	ALCOR-LanderCapsule = 100, Landing
	ApolloLEM = 100, Landing		  // 500 (200 after other training)
		bluedog-LEM-Ascent-Cockpit = ApolloLEM
		FASALM-AscentStage = ApolloLEM
		MEMLander = ApolloLEM
	landerCabinMedium = 100, Landing
	mk2LanderCabin = 100, Landing
	rn-lk-lander = 100, Landing

```

**Comment by siimav** [comment]
@glilienthal The duplicate landing courses that you saw should be fixed by #912

**Comment by pap1723** [comment]
@glilienthal has this been fixed now with #912 or is this still pending?
